packages/wallet-sdk/src/sign/walletlink/relay/WalletLinkRelay.ts

```diff
@@ -170,24 +166,10 @@ export class WalletLinkRelay implements WalletLinkConnectionUpdateListener {
          */
         const storedSession = WalletLinkSession.load(this.storage);
         if (storedSession?.id === this._session.id) {
-          this.storage.clear();
-        }
-
-        if (this._reloadOnDisconnect) {
```

Comment:
this was expected to be always true
https://github.com/coinbase/coinbase-wallet-sdk/blob/v3.9.0/packages/wallet-sdk/src/CoinbaseWalletSDK.ts#L77C1-L78C1
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/4ff8a1b5-7772-49f2-9846-07415c9a0d85)